user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -0,0 +1,227 @@
+- If you make the popup/panel max 35% of the width the labels of the form will be shown above the inputs, which might be easier to read.
+
+A good example of a DOM Form with a stepper component inside a panel with save and close button:
+
+![Example DOM Form](~/user-guide/images/LCA_Style_Guide_Form_Good.png)
+
+## Tables
+
+- Tables should only contain the most relevant data, try to limit the number of columns.
+- It's generally good practice to add a title to the table component (via Layout > Title) to indicate what the content of the table is.
+- Don't show guids in a table column, even if this data is part of your GQI query, it should be hidden in the table by using a filter on the columns that are shown.
+- Be consistent when using tables in your app(s), they should all have similar look and feel.
+- Use a visual indication (using table templates) on first column to show clearly what row is selected:  
+  ![Good Table Actions](~/user-guide/images/LCA_Style_Guide_Table_Select.gif)
+- For adding actions (e.g. edit the row) you can add icons to the first column or as the last column. The disadvantage of using the last column is that the table quick filter often blocks the access to the last column of the first row of a table. As long as that issue exists it is advised to use the first column. Use a context menu (...) to hide secondary actions. 
+
+  | 👍 Good | 👎 Bad |
+  |------|-----|
+  | ![Good Table Actions](~/user-guide/images/LCA_Style_Guide_Table_Actions_Good.png) | ![Bad Table Actions](~/user-guide/images/LCA_Style_Guide_Table_Actions_Bad.png) |
+
+- Use the table templates on each column 
+  - to add icons to each columns (where relevant)
+  - to increase the row height. The default row height is a bit small, increase to about 46px for good readability. 
+    
+    See example of a good visualization of a table below:
+
+    ![Good Table](~/user-guide/images/LCA_Style_Guide_Table_Good.png)
+
+## Grids
+
+Displaying multiple records in a LCA can be done by using a grid or a table. In most cases a table makes more sense as you have sorting and filtering options out of the box. Only when very specific visuals are needed with horizontal and/or vertical rows, it makes more sense to use a grid. Here are some examples 
+
+- Use a grid component to create a button panel.
+
+![Control Surface](~/user-guide/images/LCA_Style_Guide_Control_Surface.png)
+
+- Use a grid for button-style filters:
+
+| Horizontal filter |
+|-------------------|
+|![Grid Filter Horizontal](~/user-guide/images/LCA_Style_Guide_Grid_Filter_Horizontal.png)|
+
+| Vertical Filter |
+|-----------------|
+|![Grid Filter Vertical](~/user-guide/images/LCA_Style_Guide_Grid_Filter_Vertical.png)|
+
+## Using templates for tables and grids
+
+When using HTML inside a grid or table template make sure you have an **outer HTML element** with an empty title attribute, otherwise the HTML will show up on hover:
```

Comment:
Suggest adding the value of the column to the title

Reply:
why ? on buttons I prefer not to have a tooltip, why would want a tooltip ?